net/s3cmd/Portfile

```diff
@@ -18,21 +17,18 @@ long_description    S3cmd lets you copy files from/to Amazon S3 \
                     of Amazon's CloudFront content delivery network.
 
 homepage            http://s3tools.org/s3cmd
-master_sites        sourceforge:s3tools
 
-checksums           rmd160  33ab4c5a85b80e7563ad0d35f9dba11641373417 \
-                    sha256  4675794f84d8744ee3d35873d180f41c7b2116895ccbe2738a9bc552e1cf214e
+github.setup        s3tools s3cmd 2.1.0 v
+revision            0
+checksums           rmd160  8ebddd2ff81d7a670d87c7b2c9500ba7d9a6dbc3 \
+                    sha256  2293f775fde77201bf8e489f20516fd594168c77897168f129e5c1a2b33b7e37
 
```

Comment:
Please also add the size checksum to the list. We now ask to include this be default as well.

Reply:
🆗

Reply:
This is a little bit of mystery for me. Apart from the size checksum, that I've already added, my macports insist telling me that my checksums are correct:

```
:debug:checksum Calculated (rmd160) is 8ebddd2ff81d7a670d87c7b2c9500ba7d9a6dbc3
:debug:checksum Correct (rmd160) checksum for s3cmd-2.1.0.tar.gz
:debug:checksum Calculated (sha256) is 2293f775fde77201bf8e489f20516fd594168c77897168f129e5c1a2b33b7e37
:debug:checksum Correct (sha256) checksum for s3cmd-2.1.0.tar.gz
...
...
:info:checksum The correct checksum line may be:
:info:checksum checksums           rmd160  8ebddd2ff81d7a670d87c7b2c9500ba7d9a6dbc3 \
:info:checksum                     sha256  2293f775fde77201bf8e489f20516fd594168c77897168f129e5c1a2b33b7e37 \
:info:checksum                     size    450078
```

And they (the 3) have nothing to do with the checksums reported in the failed builds above.

Reply:
And the new checks have failed again (checksums). I really don't get why itf builds ok locally and checksums match. Do I've to clean/reset anything to get things rebuilt from scratch? I've already cleaned and issued a `-vst install`, running out of ideas.

Reply:
Try

```
sudo port clean -all s3cmd
```

Reply:
B.t.w `all` option might need two dashes... not sure and cannot check right now.

Reply:
checksums also failed for me here local, when building the update in this branch. So I am sure the issue is your tarballs are out of sync (perhaps a stealth update since you first got them).

I've pushed the necessary updates here myself.

Reply:
Thanks!